plugins/com.google.cloud.tools.eclipse.appengine.deploy/META-INF/MANIFEST.MF

```diff
@@ -21,4 +21,5 @@ Require-Bundle: com.google.cloud.tools.app.lib;bundle-version="0.1.0",
  org.eclipse.wst.common.modulecore,
  org.eclipse.ui.console,
  com.google.cloud.tools.eclipse.sdk,
- org.eclipse.e4.core.contexts
+ org.eclipse.e4.core.contexts,
+ com.google.cloud.tools.eclipse.util
```

Comment:
That shouldn't matter if the same class doesn't appear in multiple jars, right? 

I also expect that we've overused Require-Bundle (mostly my fault). We should probably be using Import-Package more in preference to Require-Bundle.